auction-server/src/api.rs

```diff
@@ -430,11 +431,15 @@ pub async fn start_api(run_options: RunOptions, store: Arc<StoreNew>) -> Result<
         }))
         .build_pair();
 
+    let original_doc = serde_json::to_value(ApiDoc::openapi())
+        .expect("Failed to serialize OpenAPI document to json value");
+
     let app: Router<()> = Router::new()
         .merge(Redoc::with_url(Route::Docs.as_ref(), ApiDoc::openapi()))
         .merge(routes)
         .route(Route::Root.as_ref(), get(root))
         .route(Route::Liveness.as_ref(), get(live))
+        .route(Route::OpenApi.as_ref(), get(original_doc.to_string()))
```

Comment:
what does this change do?

Reply:
this adds back the openapi types which weren't accessible before